lib/HLSL/DxilGenerationPass.cpp

```diff
@@ -55,19 +55,13 @@ void SimplifyGlobalSymbol(GlobalVariable *GV) {
         Function *F = LI->getParent()->getParent();
         auto it = handleMapOnFunction.find(F);
         if (it == handleMapOnFunction.end()) {
+          LI->moveBefore(dxilutil::FindAllocaInsertionPt(F));
```

Comment:
Rather than recreate an identical instruction, this relocates it. Since it's loading a global, it's fine to have it at the top.